content/docs/command-reference/add.md

```diff
@@ -169,6 +169,10 @@ not.
   [remote storage](/doc/command-reference/remote) to transfer external target to
   (can only be used with `--to-remote`).
 
+- `-j <number>`, `--jobs <number>` - parallelism level for DVC to transfer data
+  when using `--to-remote`. The default value is `4 \* cpu_count()`. For SSH
+  remotes, the default is `4`. Using more jobs may speed up the operation.
```

Comment:
I wonder if `--jobs` also has this same effect in `import-url --to-remote`. [Currently](https://dvc.org/doc/command-reference/import-url#-j) it only refers to downloading the target data. @dberenbaum or @efiop, do you know? Thanks

Reply:
`--jobs` affects the `uploading` when using `dvc import-url --to-remote` so the description should be updated. This also affects `dvc update`

Reply:
> --jobs affects the uploading when using dvc import-url --to-remote

AFAIK `--to-remote` "streams" chunks of the data via the local env into the remote. Do you mean that `--jobs` affects this whole transferring strategy (both download and upload) but has no effect for downloading regular imports (no `--to-remote`) @daavoo ? Thanks

> If you're not sure please lmk and we'll check with someone else in the core team.

Reply:
Moved this Q to https://github.com/iterative/dvc.org/issues/3233